tests/test_dump_query.py

```diff
@@ -0,0 +1,22 @@
+import lkml
+
+
+def test_dump_query():
+    # adapted from https://github.com/joshtemple/lkml/pull/55/files
```

Comment:
Do we actually need this? Hopefully we can trust lkml not to break.

Reply:
i can drop this. I wanted to test that the fix is present as we remove the workaround.